src/common/playlists/PlayThumbnail.jsx

```diff
@@ -54,9 +67,16 @@ const PlayThumbnail = ({ play }) => {
     }
   }, [play]);
 
+  const countLikes = (Obj) => {
```

Comment:
MINOR: I see this method is repeated on multiple files. So we should create a util service and put this method inside. The method name could be `countByProp(obj, propName, propValue)` and call it as 

```javascript
countByProp(Obj, 'liked', true)
```